src/Tribe/Aggregator/Service.php

```diff
@@ -526,6 +526,36 @@ public function post_import( $args ) {
 		return $this->post( 'import', $args );
 	}
 
+	public function update_import( $import_id, $args ) {
+		$api = $this->api();
+
+		// if the user doesn't have a license key, don't bother hitting the service
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: ac4c751de21ae3443f71dfeac2d850c1</sub>